examples/gcp/postgresql/main.tf

```diff
@@ -15,6 +15,6 @@ module "postgresql" {
   user_can_create_db     = true
   databases              = ["test"]
   replication            = true
-  provision_read_replica = true
+  provision_read_replica = false
```

Comment:
I don't think we want to disable read replica creation. This is where we test whether this flag works or not